examples/asr_rnnt_emformer/handler.py

```diff
@@ -0,0 +1,77 @@
+import json
+import os
+import tempfile
+
+import torch
+import torchaudio
+from torchaudio.pipelines import EMFORMER_RNNT_BASE_LIBRISPEECH
+
+
+class ModelHandler(object):
+    """
+    A custom model handler implementation.
+    """
+    def __init__(self):
+        self._context = None
+        self.initialized = False
+        self.model = None
+        self.device = None
+
+    def initialize(self, context):
+        """
+        Invoke by torchserve for loading a model
+        :param context: context contains model server system properties
+        :return:
+        """
+
+        #  load the model
+        self.manifest = context.manifest
+
+        properties = context.system_properties
+        model_dir = properties.get("model_dir")
+        self.device = torch.device("cuda:" + str(properties.get("gpu_id")) if torch.cuda.is_available() else "cpu")
+
+        # Read model serialize/pt file
+        serialized_file = self.manifest['model']['serializedFile']
+        model_pt_path = os.path.join(model_dir, serialized_file)
+        if not os.path.isfile(model_pt_path):
+            raise RuntimeError("Missing the model.pt file")
+
+        self.model = torch.jit.load(model_pt_path)
+
+        self.initialized = True
+
+
+    def handle(self, data, context):
+        """
+        Invoke by TorchServe for prediction request.
+        Do pre-processing of data, prediction using model and postprocessing of prediciton output
+        :param data: Input data for prediction
+        :param context: Initial context contains model server system properties.
+        :return: prediction output
+        """
+        if isinstance(data, list):
+            data = data[0]
+        data = data.get("data") or data.get("body")
```

Comment:
This is not able to handle batching. ref [example](https://github.com/pytorch/serve/blob/master/ts/torch_handler/vision_handler.py#L38)

Reply:
To be fair neither does our default handler , for an example I think this is fine

Reply:
For audio models, batching may not make sense in all cases. Will be good to mention as a note in the readme that this example does not support batching and verify the config properties match